### DISCUSSION TITLE
problem in making fw for fb7590 7.50+

### DISCUSSION THREAD

**Discussion by wschleifer** [Freetz-NG/freetz-ng#814] [Allgemein] [answered]
Hello, 
when building a firmware for 7590 7.50+ the following problem occurs:
![grafik](https://github.com/Freetz-NG/freetz-ng/assets/19783771/1801fe1f-8cd3-4e50-bca1-8d1676d2aa0b)

What can I do? What is the problem?

Thanks,
Werner.

**Comment by wschleifer** [comment]
a build for fb 7590 7.2 is possible without errors.

**Comment by fda77** [comment, marked as answer]
Seems the file is corrups, delete it in `dl/` to redownload

**Reply by wschleifer** [reply]
thanks a lot. This is the reason. Now it works
